app/models.py

```diff
@@ -94,3 +95,31 @@ def facet_plural(facet: str):
         facet_plural = facet
 
     return facet_plural
+
+
+class ElementsItem(BaseModel):
+    element_type: str
+    text_element: Union[str, None] = None
+
+
+class KnowledgePanelItem(BaseModel):
+    title: str
+    subtitle: Union[str, None] = None
+    source_url: Union[str, None] = None
+    elements: Union[list[ElementsItem], None] = None
+
+
+class HungerGameResponse(BaseModel):
+    hunger_game: KnowledgePanelItem
+
+
+class DataQualityResponse(BaseModel):
+    Quality: KnowledgePanelItem
+
+
+class LastEditsResponse(BaseModel):
+    LastEdits: KnowledgePanelItem
+
+
+class FacetResponse(BaseModel):
+    knowledge_panels: Union[list[DataQualityResponse], None] = None
```

Comment:
How can I add `HungerGameResponse` and `LastEditsResponse` in `class FacetResponse(BaseModel):`

Reply:
See https://fastapi.tiangolo.com/tutorial/body-nested-models/#nested-models, it's just about type annotation.


```suggestion
    KnowledgePanel = Union[DataQualityResponse, HungerGameResponse, LastEditsResponse]
    knowledge_panels: Union[list[KnowledgePanel], None] = None
```

Reply:
Thank you!